srcpkgs/datovka/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'datovka'
+pkgname=datovka
+version=4.21.1
+revision=1
+archs="*-musl"
+build_style=qmake
+hostmakedepends="pkg-config qt5-qmake qt5-host-tools"
+makedepends="libdatovka-devel qt5-websockets-devel qt5-svg-devel libxml2-devel"
+depends="libdatovka hicolor-icon-theme qt5-plugin-sqlite qt5-translations"
```

Comment:
```suggestion
depends="hicolor-icon-theme qt5-plugin-sqlite qt5-translations"
```

xbps-src adds it automatically when generating the package.

Reply:
Alright. It is a runtime dependency as well.